crypto/vrf.go

```diff
@@ -3,7 +3,7 @@ package crypto
 import (
 	"crypto/rand"
 	"errors"
-	"golang.org/x/crypto/ed25519"
+	"github.com/spacemeshos/ed25519"
```

Comment:
this isn't really related to this PR. I would appreciate if we can extract to a different pr and I'd like @barakshani to take a look. also I think @gavraz was involved so he might have good input.